src/gui/PatternsManagerWindow.cpp

```diff
@@ -74,12 +106,12 @@ void PatternsManagerWindow::create_pattern()
 
     const auto base_pfs = to_u8string(base_pfs_qs);
 
-    auto all_pfs            = settings_.current_profile().per_file_settings();
     PerFileSettings new_pfs = **std::ranges::find(all_pfs, base_pfs, [](const PerFileSettings *pfs) {
         return pfs->pattern.text();
     });
 
-    new_pfs.pattern = Pattern{base_pfs};
+    // TODO: add way to select regex or glob
```

Comment:
Not sure what you mean. We can select it now, do you mean changing?

Reply:
Forgot to remove it. 😅